posting/views.py

```diff
@@ -1,3 +1,51 @@
-from django.shortcuts import render
+import json
+import math
+from django.core.serializers.json import Serializer
 
-# Create your views here.
+from django.http           import JsonResponse, HttpResponse
+from django.views          import View
+from django.core.paginator import Paginator
+
+from posting.models   import Posting, Like, BookMark
+from utils            import non_user_accept_decorator
+
+class PostingDetailView(View):
+    @non_user_accept_decorator
+    def get(self, request, posting_id):
+        try:
+            user     = request.user
+            posting  = Posting.objects.get(id=posting_id)
+            result   = []
+            contents = {
+                    'id'           : posting.id,
+                    'title'        : posting.title,
+                    'company'      : posting.company_detail.company.name,
+                    'city'         : posting.company_detail.state.name,
+                    'district'     : posting.company_detail.county.name,
+                    'detailAddress': posting.company_detail.address,
+                    'latitude'     : float(posting.company_detail.latitude),
+                    'longitude'    : float(posting.company_detail.longitude),
+                    'tags'         : [tag_detail.name for tag_detail in posting.company_detail.company.tag.all()],
+                    'description'  : posting.description,
+                    'image'        : posting.company_detail.company.companyimage_set.first().image_url,
+                    'bonus'        : int(posting.reward),
+                    'like'         : posting.like_set.count(),
+                    'deadline'     : str(posting.end_date)[:10],
+                    'logoSrc'      : posting.company_detail.company.icon,
+                    'category'     : posting.job_category.occupation.name,
+                        }
+            if user == None:
```

Comment:
```suggestion
            if user is None:
```
None을 판별할 때는 is를 사용합니다.

Reply:
두가지 모두 수정해서 올리겠습니다!